manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -269,28 +269,17 @@ func (r *FybrikApplicationReconciler) deleteExternalResources(applicationContext
 
 // setReadModulesEndpoints populates the ReadEndpointsMap map in the status of the fybrikapplication
 // Current implementation assumes there is only one cluster with read modules (which is the same cluster the user's workload)
```

Comment:
I don't think this comment is still true.  Also need to update the comment on line 56.

Reply:
Fixed